messaging/src/test/java/org/axonframework/common/infra/JacksonComponentDescriptorTest.java

```diff
@@ -0,0 +1,423 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.common.infra;
+
+import jakarta.annotation.Nonnull;
+import org.junit.jupiter.api.*;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JacksonComponentDescriptorTest extends ComponentDescriptorTestSuite {
+
+    @Override
+    ComponentDescriptor testSubject() {
+        return new JacksonComponentDescriptor();
+    }
+
+    @BeforeEach
+    void setUp() {
+        testSubject = new JacksonComponentDescriptor();
+    }
+
+    @Override
+    protected void assertDescribeNullString(String result) {
```

Comment:
Curious: Why do some of these test methods have the `protected` modifier?

Reply:
Good question, removed :)